qiskit_ibm_runtime/accounts/management.py

```diff
@@ -26,11 +26,10 @@
 )
 
 _DEFAULT_ACCOUNT_NAME = "default"
-_DEFAULT_ACCOUNT_NAME_IBM_QUANTUM = "default-ibm-quantum"
 _DEFAULT_ACCOUNT_NAME_IBM_CLOUD = "default-ibm-cloud"
 _DEFAULT_ACCOUNT_NAME_IBM_QUANTUM_PLATFORM = "default-ibm-quantum-platform"
 _DEFAULT_CHANNEL_TYPE: ChannelType = "ibm_quantum_platform"
-_CHANNEL_TYPES = [_DEFAULT_CHANNEL_TYPE, "ibm_cloud", "ibm_quantum"]
+_CHANNEL_TYPES = [_DEFAULT_CHANNEL_TYPE, "ibm_cloud"]
```

Comment:
Comment not necessarily about this line: When I was testing the PR introducing the new channel (`ibm_quantum_platform`), I had other open projects that used an older version of runtime, and having an unknown channel name in the file would often break both service initialization and, if I'm not mistaken, even `QiskitRuntimeService.save_account`. So I had to manually delete `~/.qiskit/qiskit-ibm.json` to be able to have a clean slate. We should make sure to have a contingency plan for users that might have authentication data saved since a long time ago. At least, make sure to test it so that we understand what will break, and document it to make sure users know what to do.